src/attributes.jl

```diff
@@ -337,6 +337,18 @@ An optimizer attribute for the string identifying the solver/optimizer.
 """
 struct SolverName <: AbstractOptimizerAttribute end
 
+"""
+    LogLevel(value::Int)
+
+An optimizer attribute for level of verbosity of the optimizer. `0` means
+no output and the meaning is solver specific for positive values.
+If a solver specific attribute controlling the output level is also set, the
+last attributes set gets supremacy over the other one.
```

Comment:
How does this rule interact with the caching optimizer? The order that the user set the attributes might not match the order that they're copied to the optimizer.

Reply:
Good point. It's kind of weird for caching optimizer to store RawParameter though

Reply:
> It's kind of weird for caching optimizer to store RawParameter though

I don't think it's weird. Won't this be necessary for users to pass RawParameters from JuMP to solvers in general?